core/server/common/src/main/java/alluxio/RpcUtils.java

```diff
@@ -61,19 +86,51 @@
     try {
       return callable.call();
     } catch (AlluxioException e) {
-      LOG.debug("Internal Alluxio error when running rpc", e);
+      LOG.debug("Internal Error: {}", callable, e);
+      throw e.toThrift();
+    } catch (IOException e) {
+      LOG.debug("I/O Error: {}", callable, e);
```

Comment:
I think it would be fine to use the same "Exception running {}" for all of these exception logging statements. The name of the exception is already being logged along with the stack trace.